docs/docs/cmd/spo/commandset/commandset-set.md

```diff
@@ -0,0 +1,71 @@
+# spo commandset set
+
+Updates a ListView Command Set on a site.
+
+## Usage
+
+```sh
+m365 spo commandset set [options]
+```
+
+## Options
+
+`-u, --webUrl <webUrl>`
+: The site to update the ListView Command Set on.
+
+`-t, --title [title]`
+: The title of the ListView Command Set to update. Specify either `title`, `id`or `clientSideComponentId`.
+
+`-i, --id [id]`
+: The id of the ListView Command Set to update. Specify either `title`, `id`or `clientSideComponentId`.
+
+`-c, --clientSideComponentId  [clientSideComponentId]`
+: The Client Side Component Id (GUID) of the ListView Command Set to update. Specify either `title`, `id`or `clientSideComponentId`.
+
+`--newTitle [newTitle]`
+: The new title of the ListView Command Set.
+
+`-l, --listType [listType]`
+: The list or library type to register the Command Set on. Allowed values `List` or `Library`.
+
+`--clientSideComponentProperties  [clientSideComponentProperties]`
+: The Client Side Component properties of the ListView Command Set.
+
+`-s, --scope [scope]`
+: The scope where to lookup the ListView Command Set: at site level or web level. Allowed values: `Site`, `Web`, `All`. Defaults to `All`.
+
+`--location [location]`
+: The location of the ListView Command Set. Allowed values `ContextMenu`, `CommandBar` or `Both`.
+
+--8<-- "docs/cmd/_global.md"
+
+## Remarks
+
+Running this command from the Windows Command Shell (cmd.exe) or PowerShell for Windows OS XP, 7, 8, 8.1 without bash installed might require additional formatting for command options that have JSON, XML or JavaScript values because the command shell treat quotes differently. For example, this is how ApplicationCustomizer user custom action can be created from the Windows cmd.exe:
```

Comment:
Same comment like in `add` command. It may seem a bit strange we are referring to custom action here